resources/home/dnanexus/generate_workbook/utils/excel.py

```
@@ -543,6 +546,117 @@ def helios_summary(self) -> None:
+        self.summary.column_dimensions['A'].width = 36
+        self.summary.column_dimensions['B'].width = 16
+        self.summary.column_dimensions['C'].width = 16
+        self.summary.column_dimensions['D'].width = 16
+
+        self.summary.merge_cells(
+            start_row=1, end_row=1, start_column=2, end_column=6)
+
+        row_count = 9
+
+        # write counts of variants
+        for sheet, vcf in zip(self.args.sheets, self.vcfs):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)